src/modules/client/components/ClientProfileCard.tsx

```diff
@@ -18,15 +18,16 @@ import ClientImageUploadDialog from '@/components/elements/input/ClientImageUplo
 import NotSpecified from '@/components/elements/NotSpecified';
 import SimpleAccordion from '@/components/elements/SimpleAccordion';
 import SimpleTable from '@/components/elements/SimpleTable';
-import ClientDobAge from '@/modules/hmis/components/ClientDobAge';
-import ClientSsn from '@/modules/hmis/components/ClientSsn';
+import { ClientSafeDobAge } from '@/modules/hmis/components/ClientDobAge';
```

Comment:
I don't think we want to have two components (`ClientSafeDobAge`/`ClientDobAge`) because we should _always_ use the safe one. Can you make that the default export, and either remove the inner component or stop exporting it from the file.

Reply:
Hmm. See my comment on https://github.com/greenriver/hmis-frontend/pull/175#discussion_r1132806511. I think the extra query component is redundant, and we should just update `ClientDobAge` to appropriately handle the case where `client.age` is present and `client.dob` is not.

Reply:
I just removed the `ClientSafeDob` component and switched to checking for nullability in the original one